src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/HttpPipelineMessage.cs

```diff
@@ -23,7 +23,7 @@ public HttpPipelineMessage(CancellationToken cancellation)
 
         public HttpPipelineRequest Request { get; set; }
 
-        public HttpPipelineResponse Response { get; set; }
+        public Response Response { get; set; }
```

Comment:
I find it inconsistent that now we have Response but HttpPipelineRequest. Should HttpPipelineRequest be renamed to Request? Otherwise the PR looks good.